internal/csi-addons/rbd/replication.go

```diff
@@ -891,6 +896,9 @@ func RemoteStatus(gmis *librbd.GlobalMirrorImageStatus) (librbd.SiteMirrorImageS
 		ss  librbd.SiteMirrorImageStatus
 		err error = librbd.ErrNotExist
 	)
+	log.UsefulLog(
```

Comment:
`%q` might not help as you are logging the details, can you please check and provide the log output?

Reply:
Log this as `TraceLog` log?